package.json

```diff
@@ -66,12 +71,14 @@
     "vite-plugin-dts": "^3.6.0",
     "vitest": "^0.34.5"
   },
-  "dependencies": {
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "vanilla-framework": "^4.3.0"
```

Comment:
Did you mean to remove vanilla from the dependencies?

Reply:
Good spot, meant to move it to peerDependencies. This worked because it's a dependency of react-components. Updated this now. Please take another look.